### ISSUE TITLE
possible to set HmIP-FALMOT-C12 without homematic thermostat?

### ISSUE THREAD

**Issue by flashfingaz** [danielperna84/pyhomematic#462]
would it be possible to set/control the FALMOT-C12 directly? from a look at the code it seems the device is currently only used as a sensor and not as an actor - correct? somewhere along the way there was a comment that such option would not be needed due to the usage of the homematic thermostats. but isn't exactly that what diy home automation is about: customize the hell out of what we get from the shelves? ;)

background:
I don't like the thermostats from homematic or any other - as they are bulky and single tasked devices. so i would love to to use some NSPanel with my home assistant install to control the FALMOT-C12 and everything else. custom firmware on the NSPanel is already a very easy thing with esphome/tasmota so the missing link seems control over the FALMOT-C12 (or any other valve thermostat thing...)

tldr: if possible, it would be really handy if the FALMOT-C12 integration could support setting the individual valves in order to use custom thermostats

thanks =)

**Comment by danielperna84** [comment]
The old integration is not being worked on anymore. Check out the new integration: https://github.com/danielperna84/hahomematic

I don't know since I don't have that device, but it's possible, that what you want to do is already available there.
